.github/workflows/ubuntu_build.yml

```diff
@@ -71,10 +71,21 @@ jobs:
         run: |
           ./build.sh
 
+      - name: 'upload release build artifacts'
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        with:
+          name: mtl-release-bin
+          path: '${{ github.workspace }}/build/*'
+
```

Comment:
@zLukas same as in the `Media-Communications-Mesh` - number of retention days must be included as possibly very big archives will be stored here. 
Future feature: The best in my opinion would be to install the result in fixed path provided as variable so that we do not store not needed junk files nor sub-steps artifacts nor debug sybmols.